starter/src/main/java/com/starter/fullstack/dao/InventoryDAO.java

```diff
@@ -1,14 +1,15 @@
 package com.starter.fullstack.dao;
 
-import com.starter.fullstack.api.Inventory;
 import java.util.List;
 import java.util.Optional;
+
```

Comment:
There's a few checkstyle errors here; use `./gradlew build` to see which lines to correct.